codegen/sdk-codegen/build.gradle.kts

```diff
@@ -225,13 +217,16 @@ task("stageSdks") {
                 from(it.outputDir)
                 into(it.sourcesDir)
                 exclude { details ->
+                    // Exclude Package.swift
+                    val fileIsPackageManifest = details.file.name.endsWith("Package.swift")
+
```

Comment:
Since smithy-swift now generates a package manifest in all cases, we simply don't copy it over to the package structure.  This allows for the removal of the related SwiftSetting.